forte/data/base_pack.py

```diff
@@ -239,8 +242,30 @@ def _deserialize(
         return pack
 
     @classmethod
-    def from_string(cls, data_content: str) -> "BasePack":
-        pack = jsonpickle.decode(data_content)
+    def from_string(
+        cls, data_content: str, json_method: str = "json"
+    ) -> "BasePack":
+        if json_method == "jsonpickle":
+            pack = jsonpickle.decode(data_content)
+        elif json_method == "json":
+
+            def object_hook(json_dict):
+                """
+                Custom object hook for JSON deserialization. It will call
+                `__setstate__` to deserialize the json content into a class
+                object.
+                """
+                if json_dict.keys() == {JSON_STATE_FIELD, JSON_CLASS_FIELD}:
+                    state = json_dict[JSON_STATE_FIELD]
+                    obj_type = get_class(json_dict[JSON_CLASS_FIELD])
+                    obj = obj_type.__new__(obj_type)
```

Comment:
I think we can directly do `obj_type(obj_type)` without calling `__new__`

Reply:
The purpose of `__new__` is to avoid invoking `__init__`. `__init__` may expect some required arguments that are not available in the serialized pack data.

Reply:
hmm, sorry for the misleading. If the current implementation works we should just keep it.